src/components/app/style.scss

```diff
@@ -0,0 +1,25 @@
+@import '~style/helpers.scss';
+
+.app {
+  position: absolute;
+  display: flex;
+  flex-direction: column;
+  top: 0;
+  left: 0;
+  width: 100%;
+  bottom: 0;
+  overflow: hidden;
+  z-index: 1;
+
+  .header {
+    flex: 0 0 auto;
+    position: relative;
+  }
+
+  .content {
+    flex: 1 1 auto;
+    contain: size layout style;
+    overflow: auto;
```

Comment:
Do we really wanna scroll on content? It usually ends up causing problems. I’d prefer scrolling on body and having sidebar/topbar `position: fixed`.

Reply:
I think we've got too much layout in the PR. If it's either-or, I'd rather we landed something that simply demonstrated that CSS could be included, and didn't otherwise worry about layout.

Layout is something we can plan & do in another PR.